Cecilifier.Web/Program.cs

```diff
@@ -11,7 +11,7 @@ public class Program
         public static void Main(string[] args)
         {
             var configurationBuilder = new ConfigurationBuilder();
-            var config = configurationBuilder.AddJsonFile($"appsettings.Production.json", optional: false).Build();
+            var config = configurationBuilder.AddJsonFile($"appsettings.json", optional: false).Build();
```

Comment:
thanks for the PR but I am afraid this is the wrong solution :(

If you check `.gitignore` you'll see that I've explicitly ignored `appsettings.Production.json` and your change would force me to rename my production file locally and risky pushing it to github

The correct solution would be to rely on [ASP.NET Core configuration](https://learn.microsoft.com/en-us/aspnet/core/fundamentals/configuration/?view=aspnetcore-6.0) but it looks like it is not working due to changes I've introduced in the code (I am not a web dev :) ).

I am planing to replace the current web app with a new one based on .NET 7.0 when it ships but I am not sure when I'll be able to work on that

For now my suggestion is to change your PR to instead of rename that file, simply check if `appsettings.Production.json` exists and use it; otherwise try `appsettings.json`. 

Does that sounds like a plan ?

Reply:
@zwcloud are you planing to implement my suggestion? Otherwise I am going to add to my task list :)

Reply:
Sure. Done and tested.